steam_desktop_updater.py

```diff
@@ -167,10 +167,7 @@ def get_installed_apps(steam_root):
     apps = []
     logging.info('Searching library folders')
     with open(os.path.join(steam_root, 'steamapps', 'libraryfolders.vdf'), 'r') as lf:
-        library_folders = vdf.load(lf)['LibraryFolders']
-        assert '0' not in library_folders
-        library_folders['0'] = steam_root
-        for k, v in library_folders.items():
+        for k, v in [("0", steam_root)] + list(vdf.load(lf)['LibraryFolders'].items()):
             if not k.isdigit():
                 continue
```

Comment:
This could be simplified a bit by building list and filtering it at the same time, e.g.
```suggestion
        for v in [steam_root] + [v for k, v in vdf.load(lf)['LibraryFolders'].items() if k.isdigit()]:
```

Reply:
Yet the line is too long this way. Maybe keep `library_folders` variable.